.github/scripts/labeler.py

```diff
@@ -0,0 +1,224 @@
+from __future__ import annotations
+
+import sys
+import glob
+import subprocess
+import os
+
+
+DRY_RUN = False
+
+
+def main(changed_files: str | None = None, merge_base_schema_files: str | None = None):
+    global DRY_RUN
+
+    pr_number = os.environ.get("GITHUB_PR_NUMBER")
+    comment_file_path = os.environ.get("CI_COMMENT_FILE")
+
+    if not comment_file_path:
+        print("CI_COMMENT_FILE not set, skipping")
+        sys.exit(1)
```

Comment:
We exit 1 for the case where we're "skipping" here, but also on line 58 if there are no existing JSON files. Logging that we're "skipping" and then exiting 1 seems confusing to me. Maybe this should log that it's failing?